Formula/publish.rb

```diff
@@ -0,0 +1,21 @@
+class Publish < Formula
+  desc "Static site generator for Swift developers"
+  homepage "https://github.com/JohnSundell/Publish"
+  url "https://github.com/JohnSundell/Publish/archive/0.5.0.tar.gz"
+  sha256 "b149a21ac21640d914e31dc0e0f4c2e24b547e2f4f268921cdc21b0a7dec0538"
+  head "https://github.com/JohnSundell/Publish.git"
+
+  depends_on :xcode => ["11.0", :build]
```

Comment:
Can this just depend on the swift formula?

Reply:
We currently don't for any other Swift depending formula. Regardless, the `swift` version we have in Homebrew is a bit old now and can't be updated anymore due to upstream issues haven't been responded to. Looks like upstream uses Swift 5 and we ship Swift 4.

Reply:
Does that mean that this formula can't be added until `brew` drops support for High Sierra? Or is that not a blocker?

Looks like there is a formula that depends on 11.0 Xcode [`molten-vk.rb`](https://github.com/homebrew/homebrew-core/tree/master/Formula/molten-vk.rb), but I don't know if that's fine or not.

Reply:
I was just wondering if it needed xcode or a specific subset of it (just swift). Because swift also works on Linux while xcode doesn't. It seems that we ship a too old swift version though.

Reply:
That's a fair point. I don't think we consider Linux at all in any of the Swift formulae.

If we ever get the Swift formula up to an acceptable state again, it's something worth considering. Though it should be consistent.